code/__defines/damage_organs.dm

```diff
@@ -89,5 +89,5 @@
 // that dealing just enough burn damage to kill the player will cause the given
 // proportion of their max blood volume to be lost
 // (e.g. 0.6 == 60% lost if 200 burn damage is taken).
-#define FLUIDLOSS_WIDE_BURN 0.6 //for burns from heat applied over a wider area, like from fire
-#define FLUIDLOSS_CONC_BURN 0.4 //for concentrated burns, like from lasers
+#define FLUIDLOSS_WIDE_BURN 0.3 //for burns from heat applied over a wider area, like from fire
+#define FLUIDLOSS_CONC_BURN 0.2 //for concentrated burns, like from lasers
```

Comment:
Don't they apply to other things, including fires and whatnot, instead of only lasers?

Reply:
Intended